### DISCUSSION TITLE
Game One +1

### DISCUSSION THREAD

**Discussion by baliebox4916** [iptv-org/community#830] [Q&A]
### Channel ID

GameOnePlus1.fr

### Notes

Some Other Channel Requests

This Time Regarding Anime

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Link?

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19816
